pkg/clusters/cleanup.go

```diff
@@ -50,169 +45,6 @@ func (c *Cleaner) AddNamespace(namespace *corev1.Namespace) {
 	c.namespaces = append(c.namespaces, namespace)
 }
 
-// DumpDiagnostics gathers a wide range of diagnostic information from the test cluster, to provide a snapshot of it
-// at a given time for offline debugging. It uses the provided context and writes the meta string to meta.txt to
-// identify the result set.
-func (c *Cleaner) DumpDiagnostics(ctx context.Context, meta string) (string, error) {
```

Comment:
WDYT about keeping it for compatibility? It could just delegate the call to the `c.cluster`.

Reply:
Good idea. I've added a wrapper that delegates the call to the underlying cluster with a deprecated note. PTAL.